sections/03_categorize.md

```diff
@@ -47,53 +47,67 @@ regards to data availability and labeling.
+feature detectors. Then the small number of labeled examples could be used for
 subsequent training. Similar strategies have been employed for EHR data where
 high-quality labeled examples are also difficult to obtain
-[@doi:10.1101/039800].
+[@tag:BeaulieuJones2016_ehr_encode].
 
 In addition to radiographic images, histology slides are also being analyzed
-with deep learning approaches. Ciresan et al.
-[@doi:10.1007/978-3-642-40763-5_51] developed one of the earliest examples,
-winning the 2012 International Conference on Pattern Recognition's Contest on
-Mitosis Detection while achieving human competitive accuracy. Their approach
-uses what has become a standard convolutional neural network architecture
-trained on public data.  In more recent work,  Wang et al.[@arxiv:1606.05718]
-analyzed stained slides to identify cancers  within slides of lymph node slices.
-The approach provided a probability map for  each slide. On this task a
-pathologist has about a 3% error rate. The  pathologist did not produce any
-false positives, but did have a number of false  negatives. Their algorithm had
-about twice the error rate of a pathologist.  However, their algorithms errors
-were not strongly correlated with the  pathologist. Theoretically, combining
-both could reduce the error rate to  under 1%. In this area, these algorithms
-may be ready to incorporate into  existing tools to aid pathologists. The
-authors' work suggests that this could  reduce the false negative rate of such
-evaluations. This theme of an ensemble  between deep learning algorithm and
-human expert may help overcome some of the  challenges presented by data
-limitations.
+with deep learning approaches. Ciresan et al. [@tag:Ciresan2013_mitosis]
+developed one of the earliest examples, winning the 2012 International
+Conference on Pattern Recognition's Contest on Mitosis Detection while
+achieving human competitive accuracy. Their approach uses what has become a
+standard convolutional neural network architecture trained on public data.
+In more recent work, Wang et al. [@tag:Wang2016_breast_cancer] analyzed stained
+slides to identify cancers within slides of lymph node slices. The approach
+provided a probability map for each slide. On this task a pathologist has about
+a 3% error rate. The pathologist did not produce any false positives, but did
+have a number of false negatives. Their algorithm had about twice the error
+rate of a pathologist. However, their algorithms errors were not strongly
+correlated with the pathologist. Theoretically, combining both could reduce
+the error rate to under 1%. In this area, these algorithms may be ready to
+incorporate into existing tools to aid pathologists. The authors' work suggests
+that this could reduce the false negative rate of such evaluations. This theme
+of an ensemble between deep learning algorithm and human expert may help
+overcome some of the challenges presented by data limitations.
```

Comment:
@agitter @cgreene I think this is an important point and might be worth emphasising more, maybe in the Discussion or Conclusions.

We touch on this theme a couple of times in Categorise, e.g.:
> We don't expect that deep learning methods will replace expert review. We expect them to complement expert review by allowing more efficient use of the costly practice of manual annotation.

But maybe we should be more explicit about it?